vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -174,16 +174,19 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.28.11-hotfix.20240712/windowszip/v1.28.11-hotfix.20240712-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.28.12/windowszip/v1.28.12-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.28.13/windowszip/v1.28.13-1int.zip",
+        "https://acs-mirror.azureedge.net/kubernetes/v1.28.14/windowszip/v1.28.14-1int.zip",
```

Comment:
Please remove v1.28.5, v1.28.9 and v1.28.10 and do same for v1.29 and v1.30

For supported minor versions: Keep 4 patch versions

Reply:
done